backend/cmd/server.go

```diff
@@ -8,9 +8,24 @@ import (
 	"github.com/headlamp-k8s/headlamp/backend/pkg/config"
 	"github.com/headlamp-k8s/headlamp/backend/pkg/kubeconfig"
 	"github.com/headlamp-k8s/headlamp/backend/pkg/logger"
+	"github.com/headlamp-k8s/headlamp/backend/pkg/plugins"
 )
 
 func main() {
+	if len(os.Args) == 2 && os.Args[1] == "--list-plugins" {
```

Comment:
```suggestion
	if len(os.Args) == 2 && os.Args[1] == "list-plugins" {
```

Reply:
This would change the option from `./headlamp-server --list-plugins` to `/headlamp-server list-plugins`, is that the intention? Do we want to make this distinct from other options @joaquimrocha

Reply:
Yes. This way, at a later stage we can add options to this subcommand (`./headlamp-server list-plugins --paths` or `./headlamp-server list-plugins --static` or `./headlamp-server list-plugins --user-added`)

Reply:
Done